client/components/LearnerViewDirective.js

```diff
@@ -249,10 +249,20 @@ tie.directive('learnerView', [function() {
         $scope.submitCode = function(code) {
           SolutionHandlerService
             .processSolutionAsync(prompts[currentPromptIndex], code, language)
-            .then(appendFeedback);
+            .then(setFeedback);
         };
 
-        clearFeedback();
+        $scope.feedbackParagraphs = [
+          'Greetings!',
+          'This set of exercises focuses on string manipulation.',
```

Comment:
Ahh -- there are two issues here, actually. On the one hand, yes, if this is question specific, it should go into the data schema ( /cc @eyurko ).  On the other hand, the text that's currently in it is at the set-of-questions level.

So this needs further fleshing out (about what data belongs at which level), and for now I think that a TODO comment is best.